RadarSDK/RadarAPIClient.m

```diff
@@ -243,6 +250,10 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
     params[@"country"] = [RadarUtils country];
     params[@"timeZoneOffset"] = [RadarUtils timeZoneOffset];
     params[@"source"] = [Radar stringForLocationSource:source];
+    if ([RadarSettings crossPlatform]) {
+        params[@"xPlatformType"] = [RadarSettings crossPlatformSDKType];
+        params[@"xPlatformSdkVersion"] = [RadarSettings crossPlatformSDKVersion];
+    }
```

Comment:
let's else a `params[@"xPlatformType"] = @("native");` to make it clear on the user